index.html

```diff
@@ -1,15 +1,36 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <title>Angular</title>
-    <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/angular-ui-router/0.2.15/angular-ui-router.min.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.5.0-beta.2/angular-resource.min.js"></script>
-
-    <script src="js/app.js"></script>
-  </head>
-  <body>
-    <h1>Grumblr</h1>
-
-  </body>
+<html data-ng-app="grumblr">
+
+<head>
+  <title>Angular</title>
+  <base href="http://localhost:8080/" />
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/angular-ui-router/0.2.15/angular-ui-router.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.5.0-beta.2/angular-resource.min.js"></script>
+
+  <script>
+    var grumbles = [{
+      title: "I am Grumble One"
+    }, {
+      title: "I'm another Grumble"
+    }]
+  </script>
+
+  <script src="js/app.js"></script>
+  <script src="js/grumbles/grumbles.module.js"></script>
+  <script src="js/grumbles/index.controller.js"></script>
+  <script src="js/grumbles/show.controller.js"></script>
+</head>
+
+<body>
+  <h1><a data-ui-sref="grumbleIndex">Grumblr</a></h1>
+  <div data-ui-view></div>
+
+  <h2>{{GrumbleShowViewModel.grumble.title}}</h2>
```

Comment:
not sure this is what we want, this code should be in the `index.html` for our grumbles, since that is where the `GrumbleShowViewModel` can be accessed